python/ray/train/tests/test_examples.py

```diff
@@ -66,30 +65,30 @@ def test_tf_non_distributed(ray_start_4_cpus):
     trainer.fit()
 
 
-# TODO: Refactor as a backend test.
-def test_tensorflow_mnist_fail(ray_start_4_cpus):
-    """Tests if tensorflow example works even with worker failure."""
-    epochs = 3
+# # TODO: Refactor as a backend test.
```

Comment:
can we actually keep these tests and rewrite them with TrainingIterator? Instead of KillCallback we can just manually kill during the for loop iteration.

Reply:
I can work on this tomorrow if you'd like

Reply:
Yeah that would be great, please do that!